pom.xml

```diff
@@ -175,6 +176,18 @@
             <version>3.4.6</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
```

Comment:
What's the motive behind using powermock? So far we'ven been okay with using just `mockito`